go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/cosmos/cosmos-sdk v0.47.4
 	github.com/cosmos/gogoproto v1.4.10
 	github.com/cosmos/ibc-go/v7 v7.2.0
+	github.com/gogo/protobuf v1.3.3
```

Comment:
We should not have `gogo/protobuf` in the `go.mod` for v47.  We need to replace all uses of it with `cosmos/gogoproto`